packages/@react-stately/color/src/Color.ts

```diff
@@ -201,3 +202,14 @@ class HSLColor implements ColorValue {
     return this;
   }
 }
+
+export function parseColor(value: ColorInput): Color {
+  if (!value) return null;
+  return new Color(typeof value === 'string' ? value : value.toString('hex'));
+}
+
+export function parseColorToInt(value: ColorInput) {
+  if (!value) return null;
+  const colorString = parseColor(value).toString('hex').substring(1);
+  return parseInt(colorString, 16);
+}
```

Comment:
Check with team if this is an appropriate place for utils like these

Reply:
Maybe they could be methods on `Color` itself? `parseColor` could be a `Color.parse` static method, and `parseColorToInt` could be a `color.toInt()` method?

Reply:
Updated! Added methods in `Color` as described.